packages/web-runtime/package.json

```diff
@@ -14,9 +14,10 @@
     "luxon": "^1.22.0",
     "moment": "^2.24.0",
     "oidc-client": "1.10.1",
-    "owncloud-design-system": "^6.3.0",
+    "owncloud-design-system": "^6.0.0",
```

Comment:
Definitely not `6.0.0` should we leave it empty/don't merge this PR until it is shure which version bump ODS gets next? (Perhaps: 6.4, not sure though)

Reply:
that's goo and won't need any further bumps till we reach version 7.
```caret (^) | ^6.0.0 | 6.*.*```

Reply:
Great! 👍🏽